src/test/java/org/opensrp/util/UtilTest.java

```diff
@@ -69,5 +70,12 @@ public void testGetXlsToJsonForInvalidXls() throws IOException, JSONException {
 		String path = getFullPath("sampleXLS/invalidXLS.xls");
 		Utils.getXlsToJson(path);
 	}
-	
+
+    @Test(expected = Test.None.class)
+    public void testCloseCloseable() {
+        final Closeable closeable = () -> {
+
```

Comment:
Why don't you pass a Mock Closeable and then verify that the close method was invoked after invoking `Utils.closeCloseable`

Reply:
Sure, done